### PR TITLE
Monica Parrillo - Fatima's Pod

### PR DESCRIPTION
**Pull request by monicaleep**
resubmtting with file name changed!
Felt comfortable with the deliverable after class yesterday (monday)

### CODE REVIEW

File: js/monicaparrillopizza.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    toppings: ['pepperoni','sausage','chicken','cheese'],
+    price: 12.99
+  },
+  vegan:{
+    toppings: ['mushroom','sauce','broccoli','sadness'],
+     price: 5.99
+  },
+  hawaiian:{
+    toppings:['bacon','ham','pineapple'],
+    price: 12.99
```
Comment (ghost) [inline js/monicaparrillopizza.js:21]: fancy pricing 😆 most people just put *integers* instead of *floats*

File: js/monicaparrillopizza.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  }
+}
+for (item in menu){
+  console.log(item);
+}
+// console.log(Object.keys(menu));
+console.log(Object.prototype.hasOwnProperty('banana'))
```
Comment (ghost) [inline js/monicaparrillopizza.js:28]: hm, when you see the `Object.prototype` thing on MDN, that usually means to put the name of the object there. so this should be `menu.hasOwnProperty('banana')`

**Comment by ghost** [comment]
nice work overall! small comment about `Object.prototype` and what it means, but other than that your code looks solid. 👍 marking complete

**Comment by monicaleep** [comment]
Thanks Fatima. I don’t know how I made that mistake I will look into it. Also the global Object doesn’t have a property banana so I didn’t catch the error since it returned false!

**Pull request closed without merging**